lib/diagnostic.js

```diff
@@ -8,21 +8,40 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
```

Comment:
This function will only return the first argument (or ```undefined``` if there are no arguments).